app/Http/Services/ApiServices/HrmsSecretKeyService.php

```
@@ -0,0 +1,238 @@
+                    "email",
+                    "email_verified_at",
+                    "password",
+                    "remember_token",
+                    "type",
+                    "accessibilities",
+                    "employee_id",
+                    "created_at",
+                    "updated_at",
+                    "deleted_at",
+                ]
+            );
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\User' in method 'syncUsers'. (undefined)

(StaticAccess)